modules/manage/partials/tiered-storage.adoc

```diff
@@ -1250,9 +1250,257 @@ endif::[]
+You can take the following steps to restore a failed cluster:
+
+. Start a new cluster 
+. Restore data from a failed source cluster to the new cluster
+. Perform a rolling restart, if necessary
+
+Use this recovery method to restore your application to the latest functional state as quickly as possible. 
+
+CAUTION: Whole cluster restore is not intended to be a fully-functional disaster recovery solution. It does not provide snapshot-style consistency: some partitions in some topics will be more up to date than others. For example, restoring a consumer offsets topic could have offsets that point ahead of the restored data, although it will reflect the latest available offset in the topic data that is being restored. Committed transactions are not guaranteed to be atomic.
+
+=== Start a target cluster
+
+For Redpanda Enterprise, follow the steps to https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/[deploy a new cluster] in a self-hosted environment.
+
+For Redpanda Cloud, follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster]. 
+Cloud account credentials (such as secrets or access keys) are not included in backed up metadata. You must manually configure these credentials for the new cluster.
+
+=== Whole cluster restore
+
+Data from a failed cluster can be restored to a target cluster using an rpk command. 
+
+[,bash]
+----
+rpk cluster storage restore start
+----
+
+The following metadata is included in the restore:
+
+- Topic definitions. If you have enabled tiered storage only for specific topics, topics without tiered storage enabled will not be restored 
+- Users and access control lists (ACLs)
+- Schemas
+- The xref:develop:consume-data/consumer-offsets.adoc[consumer offsets topic]
+- All cluster configurations, including your Redpanda license key
+- Transaction metadata, up to the highest committed transaction. In-flight transactions are treated as aborted and will not be included in the restore
+
+NOTE: If you need to restore only a subset of topic data, consider using <<remote-recovery,remote recovery>> instead of a whole cluster restore.
+
+The destination cluster must be empty; that is, the cluster cannot contain data other than node and cluster configuration properties. If metadata indicating usage is found in the destination (such as user-provided schemas, non-superusers, ACLs, consumer offsets, or user-created topics), the restore will fail and the rpk command will issue a warning.
+
+If duplicate metadata is found in the destination cluster, it will be overwritten by the restore.
+
+Whole cluster restore supports only one source cluster. It is not possible to consolidate multiple clusters into the target cluster.
+
+=== Perform rolling restart
```

Comment:
This sounds good to me actually, any thoughts @mattschumpert ? I'd create a separate issue and PR for this, and it would be a blocker for this PR cause I'd need to merge that new one first before I can incorporate the new section in this PR. But it shouldn't take as long.

Reply:
Do you mean the Rolling restart section?  I have no idea why a rolling restart is being recommended in this doc.  It seems misplaced, right @andrwng ?

Reply:
It's sometimes warranted (e.g. to ensure the cluster reflects newly restored configs)